chaosk8s/actions.py

```diff
@@ -121,3 +122,33 @@ def scale_microservice(name: str, replicas: int, ns: str = "default",
         raise ActivityFailed(
             "failed to scale '{s}' to {r} replicas: {e}".format(
                 s=name, r=replicas, e=str(e)))
+
+
+def kill_statefulset(name: str, ns: str = "default",
+                     label_selector: str = "name in ({name})",
```

Comment:
The default label should be set to `None` from now on. The pattern here is legacy and probably won't work in recent k8s as the name is set to the metadata itself.

So, please set the label_selector to None by default, then, if you want, add a `name: str = None` property but you'll need to use the `field_selector` like this: https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/4bc14bdfb9111fbfa2ac6286ea6490bd05b86121/chaosk8s/node/actions.py#L50 (you can even passa field_selector instead of name as an argument so that we can filter by any other field beside name).